src/EPR.RegulatorService.Frontend.Web/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,7 @@
+namespace EPR.RegulatorService.Frontend.Web.Extensions
+{
+    public static class DateTimeExtensions
+    {
+        public static DateTime UtcToGmt(this DateTime dateTime) => TimeZoneInfo.ConvertTimeFromUtc(dateTime, TimeZoneInfo.FindSystemTimeZoneById("Europe/London"));
```

Comment:
Please see comment here: https://github.com/DEFRA/epr-frontend-accountcreation-microservice/pull/22/files/443402edf5bfbf84963c07c58507c37956fa291b#r1774845747

Reply:
They are Linux boxes and this is the same code already used in the producer frontend. I am simply extending/copying it over to the other frontends that never had to do timezone conversion